.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: go
 go:
-  - 1.5
+  - 1.7
```

Comment:
done. I also added #124 to these PRs because I needed to change the CC env anyway.